<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>78</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QLabel" name="status_label">
     <property name="styleSheet">
      <string notr="true">color: #888; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>消息过滤</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="filter_0">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_1">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_3">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>3</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_4">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_5">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_6">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_7">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: grey;
    color: black;
    border: none;
    padding: 2px;  /* 较小的内边距 */
    border-radius: 4px;
    min-width: 20px;   /* ← 关键：设置最小宽度 */
    min-height: 20px;  /* 可选：固定最小高度，防止变化 */
}
QPushButton:checked {
    border: 2px solid black;  /* 选中时的黑色边框 */
    border-radius: 4px;
	padding: 0px;
}</string>
          </property>
          <property name="text">
           <string>7</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_all">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: black;
    border: 1px solid #c0c0c0;
    padding: 2px;
    border-radius: 4px;
    min-width: 20px;
    min-height: 20px;
}

QPushButton:hover {
    background-color: #dcdcdc;
    border: 1px solid #a0a0a0;
}

QPushButton:pressed {
    background-color: #c0c0c0;
    padding-top: 5px;      /* 按下时“下沉”效果 */
    padding-bottom: 3px;
}</string>
          </property>
          <property name="text">
           <string>A</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_none">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #f0f0f0;
    color: black;
    border: 1px solid #c0c0c0;
    padding: 2px;
    border-radius: 4px;
    min-width: 20px;
    min-height: 20px;
}

QPushButton:hover {
    background-color: #dcdcdc;
    border: 1px solid #a0a0a0;
}

QPushButton:pressed {
    background-color: #c0c0c0;
    padding-top: 5px;      /* 按下时“下沉”效果 */
    padding-bottom: 3px;
}</string>
          </property>
          <property name="text">
           <string>N</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
